### PR TITLE
Completed Task 850 : Design System > Element > Progress Bar > Enhancements in style for Circular

### LINKED ISSUE #850: Design System > Element > Progress Bar > Enhancements in style for Circular
Design System > Element > Progress Bar > Enhancements in style for 

Circular
Line
Stepper
Dash
Block

Image

### PR DESCRIPTION
**Pull request by rohit9096**
## Description: 

> Design System > Element > Progress Bar > Circular
> Design System > Element > Stepper > Stepper
> Design System > Element > Stepper > Dash
> Design System > Element > Stepper > Block

## Screenshots
block
circular
dash
stepper
:

### CODE REVIEW

File: raaghu-elements/src/rds-progress-bar/rds-progress-bar.tsx [hunk 1]
```diff
@@ -12,6 +12,8 @@ export interface RdsProgressBarProps {
     displayLabel?: boolean;
     displayPercentage?: boolean;
     width?: string;
+    State?: string;
+    state?: string;
```
Comment (hemalatagaikwad) [inline raaghu-elements/src/rds-progress-bar/rds-progress-bar.tsx]: remove unused props

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]

File: raaghu-elements/src/rds-progress-bar/rds-progress-bar.tsx (hunk 1 above)
Comment (rohit9096) [reply raaghu-elements/src/rds-progress-bar/rds-progress-bar.tsx]: @hemalatagaikwad these props are used to change the state for default, inporgress, completed and error we cant remove that 

image

**Review by hemalatagaikwad — CHANGES_REQUESTED** [review]
please resolved conflicts
image

**Review by hemalatagaikwad — APPROVED** [review]
review and approve

**Review by hemalatagaikwad — APPROVED** [review]
review and approve

**Pull request merged**